src/Cms/App.php

```diff
@@ -464,29 +464,26 @@ public function controller(
 		$data = [];
 
 		// always use the site controller as defaults, if available
-		if ($controller = $this->controllerLookup('site')) {
-			$data = (array)$controller->call($this, $arguments);
+		$site   = $this->controllerLookup('site', $contentType);
+		$site ??= $this->controllerLookup('site');
```

Comment:
`Discuss:` Is it ok that we use either or? Or should both site controllers be used, if available?

Reply:
We used either/or so far and I think we should stick with that.